src/mono/mono/mini/wasm_m2n_invoke.g.h

```diff
@@ -296,6 +296,16 @@ wasm_invoke_iiliiii (void *target_func, InterpMethodArguments *margs)
 
 }
 
+static void
+wasm_invoke_iiiliiii (void *target_func, InterpMethodArguments *margs)
```

Comment:
What does "iiiliiii" mean?

Reply:
Oh, it's the signature, i.e. it returns an int, and accepts two ints, then a long, then four ints?

Reply:
correct